src/tools/tidy/src/edition.rs

```diff
@@ -1,10 +1,15 @@
-//! Tidy check to ensure that crate `edition` is '2018'
+//! Tidy check to ensure that crate `edition` is '2018' or '2021'.
 
 use std::path::Path;
 
 fn is_edition_2018(mut line: &str) -> bool {
     line = line.trim();
-    line == "edition = \"2018\"" || line == "edition = \'2018\'"
+    line == "edition = \"2018\""
```

Comment:
Thanks for cleaning this up to only allow `"`; I was thinking of doing that but didn't want to churn other crates in my PR.